components/clarity-lsp/src/vscode_bridge.rs

```diff
@@ -26,7 +26,6 @@ use std::sync::{Arc, RwLock};
 use wasm_bindgen::prelude::*;
 use wasm_bindgen_futures::future_to_promise;
 
-#[cfg(debug_assertions)]
```

Comment:
Not that `println!` won't be visible in wasm. 

I just removed `#[cfg(debug_assertions)]` for debugging, don't remember how to enable it